src/main/webapp/app/exercises/file-upload/manage/file-upload-exercise-update.component.ts

```diff
@@ -72,10 +72,19 @@ export class FileUploadExerciseUpdateComponent implements OnInit {
         window.history.back();
     }
 
+    /**
+     * Sanitize attributes of `fileUploadExercise`
+     */
+    sanitizeFileUploadExercise() {
+        this.fileUploadExercise.title = this.fileUploadExercise.title?.trim();
+    }
```

Comment:
What about moving this as a static method into `Exercise`?

Reply:
I added a static method `sanitize()` to `Exercise`.